qp_deblur/deblur.py

```diff
@@ -373,6 +373,9 @@ def deblur(qclient, job_id, parameters, out_dir):
     prep_info = qclient.get(
         '/qiita_db/prep_template/%s/' % artifact_info['prep_information'][0])
     df = pd.read_csv(prep_info['prep-file'], sep='\t')
+    if prep_info['data_type'] not in {'16S', '18S', 'ITS'}:
+        error_msg = ('deblur was developed only for target gene data')
```

Comment:
is `target gene data` a good name? How about "amplicon sequencing data" it might be closer to the wet lab vocabulary